lib/phlex/configuration.rb

```diff
@@ -2,6 +2,10 @@
 
 module Phlex
 	class Configuration
-		# Config coming soon.
+		attr_accessor :phlex_path
```

Comment:
I was in doubt about that name or `views_path`. If you have some suggestion I will accept it.